### ISSUE TITLE
make trashcan.txt also disallow login

### ISSUE THREAD

**Issue by wwiv** [wwivbbs/wwiv#766]
ideally we'd dump a caller once a name in the trashcan is found.  Right now it's only used by newuser name checking.

From WWIVDEV:

```
MM> I think Root should be an automatically rejected name to logon to get rid
MM> of the script kiddies a bit quicker.

Good idea.

Once someone types in ROOT, it should just dump them.  I have seen that happen
over here, too.  The nice thing is my firewall won't allow more than one telnet
connection to my setup from a single IP address at a time.

Basically, they can't busy out all my nodes from the same IP.

I should write a mod for my 4.24a setup to do just that.

- Mark                            
```

**Comment by wwiv** [comment]
fixed in https://github.com/wwivbbs/wwiv/commit/c442eb83906ec4bbb0d76b026f72264ffe3d545f
